### ISSUE TITLE
Formatted output doesn't parse

### ISSUE THREAD

**Issue by youshy** [tealeg/xlsx#472]
I have written a small app for copying xlsm files to html tables. Whenever there's any formatting in the xlsm file involved, it won't parse correctly.

Code:

```golang
var htmlFile []string
for _, sheet := range xlFile.Sheets {
		var htmlString string
		htmlString += "<!DOCTYPE html><html><head></head><body>"
		htmlString += "<table>"
		for _, row := range sheet.Rows {
			htmlString += "<tr>"
			for _, cell := range row.Cells {
				cellText := cell.String()
				htmlString += "<td>" + cellText + "</td>"
			}
			htmlString += "</tr>"
		}
		htmlString += "</table></body></html>"
		htmlFile = append(htmlFile, htmlString)
	}
```

Output from excel:

|input|output from excel|output in html table|
|---|---|---|
|18/03/1989|18 March 1989 (format of *Wednesday, 14 March 2012, language: English (UK))|32585|
|17:15:00|05:15:00 PM (format of 01:30:55 PM, language: English (UK))|0.71875|

Basically, any date and time format.

**Comment by tealeg** [comment]
@youshy the most likely cause is that your cells are set to be Date types, and not simply strings with date formatting - that means you'll have to use `cell.GetTime` to get a native Go `time.Time` and handle the formatting yourself.

Sorry for the late response, I'm going to close this because it's old, and probably not bug, per se.  If you want to reopen this and explore more deeply please feel free to do so.
